index.html

```diff
@@ -0,0 +1,167 @@
+      <p>
+         I can help you build a product , feature or website Look through
+         some of my work and experience! If you like what you see and 
+         have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <ul class="list-items">
+        <li><a href=""><img src="images/twitter.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/Linkedin-icon.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/medium.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/github.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/angellist.svg" alt="social-icons"></a></li>
+      </ul>
+        <ul>
+          <li><a href=""><img src="images/Disabled.svg" alt="disabled"></a></li>
+        </ul>
+          <h1>My Recent <br> Works</h1>
+        <ul>
+          <li><a href=""><img src="images/Indicator.svg" alt="indicator"></a></li>
+        </ul>
+    </section>
+    <div class="project">
+      <div class="grid container">
+        <h2>Multi-Post Stories <br> Gain+Glory</h2>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
+    <div class="project">
+      <div class="grid container">
+        <h2>Multi-Post Stories <br> Gain+Glory</h2>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
+    <div class="project">
+      <div class="grid container">
+        <h2>Multi-Post Stories <br> Gain+Glory</h2>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
+    <div class="project">
+      <div class="grid container">
+        <h2>Multi-Post Stories <br> Gain+Glory</h2>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
+    <div class="project">
+      <div class="grid container">
+        <h2>Multi-Post Stories <br> Gain+Glory</h2>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
+    <div class="project">
+      <div class="grid container">
+        <h2>Multi-Post Stories <br> Gain+Glory</h2>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
+    <div class="aboutme">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_19](https://user-images.githubusercontent.com/69638013/159370063-1b8ca58e-3bd0-4784-baa6-bc32a0308700.png)  |  ![Screenshot_18](https://user-images.githubusercontent.com/69638013/159370264-2b8b717c-95f1-47a5-898a-35ff2e24e04c.png)

- [ ]  Please add the right background image, this will make your design look like the main design

- [ ] Kindly use the right color for the button "get resume" add the button styling: Active, Hover, and Pressed.
- check the screenshot below  
![sssaa](https://user-images.githubusercontent.com/69638013/159368002-cd7b9fec-9ecc-4718-aea5-2da598790753.png)

-Just as the previous  reviewer requested make your design look like the figma design.

### OPTIONAL 

- [ ] Kindly change this div tag to section tag as this is wrapping all the about section

Reply:
Hi @ppmarq1  please check your design and I advise you make your design look like the original, please check screenshot  below and remove all borders for button